connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnectorTest.java

```diff
@@ -59,12 +59,10 @@ public void testMirrorCheckpointConnectorDisabled() {
 
         Set<String> knownConsumerGroups = new HashSet<>();
         knownConsumerGroups.add(CONSUMER_GROUP);
+
         // MirrorCheckpointConnector as minimum to run taskConfig()
-        MirrorCheckpointConnector connector = new MirrorCheckpointConnector(knownConsumerGroups,
-                                                                            config);
-        List<Map<String, String>> output = connector.taskConfigs(1);
         // expect no task will be created
-        assertEquals(0, output.size(), "MirrorCheckpointConnector not disabled");
+        assertEquals(0, new MirrorCheckpointConnector(knownConsumerGroups, config).taskConfigs(1).size(), "MirrorCheckpointConnector not disabled");
```

Comment:
nit: As far as I can tell this just inlined the calls. I think I preferred the previous format so it stays consistent with the other tests, and we avoid this really long line.